packages/app-toolbox/src/Rpc/Selection.tsx

```diff
@@ -139,7 +79,7 @@ class Selection extends TxComponent<Props, State> {
         const { rpc = prevState.rpc, accountId = prevState.accountId, values = prevState.values } = newState;
         const reqCount = rpc.params.reduce((count, { isOptional }): number => count + (isOptional ? 0 : 1), 0);
         const isValid = values.reduce((isValid, value): boolean => {
-          return isValid && value.isValid === true;
+          return isValid && value.isValid;
```

Comment:
why are we removing strict comparison here?

Reply:
webstorm tells me it's redundant haha